### PR TITLE
refactor: edit wallet in neuron-wallet

### PR DESCRIPTION
**Pull request by BaojunCZ**

### CODE REVIEW

File: packages/neuron-wallet/src/controllers/wallets.ts [hunk 1]
```diff
@@ -15,6 +15,8 @@ export enum WalletsMethod {
   Delete = 'delete',
   GetActive = 'getActive',
   Activate = 'activate',
+  RenameWallet = 'renameWallet',
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/controllers/wallets.ts]: `rename` and `update addresses` belong to `update`

Comment (BaojunCZ) [reply packages/neuron-wallet/src/controllers/wallets.ts]: fixed

File: packages/neuron-wallet/src/services/wallets.ts [hunk 2]
```diff
@@ -34,33 +34,19 @@ export default class WalletService {
...
-  //       wallet.address = address
-  //     }
-  //     if (publicKey) {
-  //       wallet.publicKey = publicKey
-  //     }
-  //     return true
-  //   }
-  //   return false
-  // }
+  public update = ({ id, name, addresses }: { id: string; name?: string; addresses?: Addresses }): boolean => {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/wallets.ts]: use object combination to update wallet, like the following:
```javascript
const currentWallet = walletStore.get(newWallet.id)
walletStore.update({...currentWallet, ...newWallet})
```

File: packages/neuron-wallet/src/services/wallets.ts [hunk 3]
```diff
@@ -34,33 +34,9 @@ export default class WalletService {
...
-  //     }
-  //     if (publicKey) {
-  //       wallet.publicKey = publicKey
-  //     }
-  //     return true
-  //   }
-  //   return false
-  // }
+  public update = (walletId: string, newWallet: WalletData) => {
+    walletStore.update(walletId, newWallet)
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/wallets.ts]: It seems that `undefined` not handled in this case.
if `newWallet.name` is undefined, it's current name should be used.

```javascript
{ ...oldWallet, ...newWallet }
```
will cover older wallet except the undefined fields.

Comment (BaojunCZ) [reply packages/neuron-wallet/src/services/wallets.ts]: get it

**Pull request merged**